src/ol/source/GeoTIFF.js

```diff
@@ -24,7 +25,8 @@ import {fromCode as unitsFromCode} from '../proj/Units.js';
 /**
  * @typedef {Object} SourceInfo
  * @property {string} url URL for the source GeoTIFF.
- * @property {Array<string>} [overviews] List of any overview URLs.
+ * @property {Blob} blob Blob containing the source GeoTIFF. `blob` and `url` are mutually exclusive.
```

Comment:
Both `url` and `blob` should now be optional.

Reply:
Adding the [ and ] brackets was enough, right? Is now part of the new PR #13724